_posts/2021-06-24-debezium-1-6-cr1-released.adoc

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title:  Debezium 1.6.0.CR1 Released
+date:   2021-06-24
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, outbox ]
+author: ccranfor
+---
+
+It's my pleasure to announce the release of Debezium *1.6.0.CR1*!
+
+This release adds skipped operations optimizations for SQL Server, introduces Heartbeat support to the Oracle connector,
+Oracle BLOB/CLOB support is now opt-in only, and provides a range of bug fixes and other improvements across different Debezium connectors.
+
++++<!-- more -->+++
+
+== Skipped operations optimizations - SQL Server
+
+Debezium introduced the notion of `skipped.operations` some time ago.
+This feature was to allow connectors to control what change events were emitted during the streaming phase, optionally omitting certain changes such as updates or deletes.
+
+The `skipped.operations` feature was originally implemented to act more like an SMT where it was a filter that was applied toward the end of the CDC pipeline just before an event was emitted.
+While this worked well, there was an optimization that could be achieved here because SQL Server records the change type in the capture table along side the change data.
+Using this knowledge, the query to fetch changes from the capture table was adjusted so that the changes returned from SQL Server are pre-filtered based on these "skipped operations".
+
+Overall, this optimization should improve both the time spent during iterations of capturing changes when the conenctor is configured to use `skipped.operations` as well as reduce the overall garbage collection that the connector may do over its lifetime due to generating fewer objects that are later discarded.
+
+Many thanks to https://github.com/blcksrx[Hossein Torabi] for identifying this optimization and contributing it!
+
+== Oracle Heartbeat Support
```

Comment:
```suggestion
== Heartbeat Support for the Oracle Connector
```

I know we often say "MySQL", "Oracle", etc. when actually referring to the "Debezium MySQL connector" or the "Debezium connector for Oracle". That's fine internally, but let's use it more clearly in public posts and the likes.